docs/program-setup/bare_metal/c_small_libs.md

```diff
@@ -56,7 +56,3 @@ After you have completed the steps above, add `small` to the `supported_c_libs`
     "iar": ["std"]
 }
 ```
```

Comment:
@hugueskamba  there's a whole explanation here about uARM in a sort of if/then method. I suppose it needs to be updated.

Reply:
Checked with Evelyne; she said it's still correct for bare metal